### ISSUE TITLE
game crash

### ISSUE THREAD

**Issue by NeoStarWonV** [MehVahdJukaar/Dummy_1.18#44]
[crash-2024-09-20_02.58.27-server.txt](https://github.com/user-attachments/files/17065144/crash-2024-09-20_02.58.27-server.txt)
version is 1.20.1

**Comment by MehVahdJukaar** [comment]
yeah you definitely need to provide more info. what did you attack it with?

**Comment by MehVahdJukaar** [comment]
Issue NOT from this mod. its from Champion mod. They are adding 2 damage types, one of which is somehow unregistered. This is an invalid condition and they must fix. you should report to them
